src/components/MergeArticleList.tsx

```diff
@@ -102,9 +98,9 @@ const MergeArticleList: React.FC<IProps> = () => {
   ]
 
   return (
-    <Container>
+    <Box as="div" mb="4rem">
```

Comment:
`Box` by default renders a `div` :)
```suggestion
    <Box mb="4rem">
```